Project-I-Social-Card/social-card/src/index.js

```diff
@@ -3,4 +3,10 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<HeaderContainer />, rootElement);
+ReactDom.render(<CardContainer />, document.getElementById('root'));
```

Comment:
Okay, this is going to be a bit of a labyrinthine explanation, but try to follow it. First of all, you don't want to render your components in the index.js. That is usually used for one purpose, and that is to select a DOM node, and render the main app file to that DOM node. Your entire app will be rendered via the single DOM node.
```suggestion

```